routes/api/hacker.js

```diff
@@ -307,6 +307,37 @@ module.exports = {
                 Middleware.Hacker.sendStatusUpdateEmail,
                 Controllers.Hacker.updatedHacker
             );
+        
+        /**
+         * @api {patch} /hacker/accept/:email accept a Hacker
+         * @apiName acceptHacker
+         * @apiGroup Hacker
+         * @apiVersion 2.0.0
+         *
+         * @apiSuccess {string} message Success message
+         * @apiSuccess {object} data Hacker object
+         * @apiSuccessExample {object} Success-Response:
+         *      {
+         *          "message": "Changed hacker information",
+         *          "data": {
+         *              "status": "Accepted"
+         *          }
+         *      }
+         * @apiPermission Administrator
+         */
+        hackerRouter
+        .route("/acceptEmail/:email")
+        .patch(
+            Middleware.Auth.ensureAuthenticated(),
+            Middleware.Auth.ensureAuthorized([Services.Hacker.findByEmail]),
+            Middleware.Validator.RouteParam.emailValidator,
+            Middleware.parseBody.middleware,
+            Middleware.Hacker.findByEmail,
+            Middleware.Hacker.parseAcceptEmail,
+            Middleware.Hacker.updateHackerByEmail,
```

Comment:
Most likely you may not even need to create this updateHackerByEmail middleware, you can have a middleware that from the email gets the hacker into the appropriate field as well as req.body.status to accepted (this could be done in parseAcceptEmail) and then just call updateHacker.